<h1><u>Exercise 1</u></h1>
It is somewhat interesting that (5 * 4 * 3 * 2 * 1) perfectly divides (10 * 9 * 8 * 7 * 6) - there's no remainder.

If we only wanted exactly four heads as opposed to five, the equivalent calculation would be (10 * 9 * 8 * 7) / (4 * 3 * 2 * 1).

Does that evenly divide too?

In [1]:
#Calculation to see if it divides evenly
(10 * 9 * 8 * 7) / (4 * 3 * 2 * 1)

210.0

The answer we get is 210 with no remainder so yes it does divide in evenly

What is the formula in general?

The general formula is called the Binomial Distribution Formula:

                P(x:n,p)= nCx(px(1-p)n-x)
n = the number of experiments (10 in this case)

x = the amount of successes(number of heads which is 5 in this case)

p = Probability of success in a single experiment (0.5 in this case)

Does it always come out as a positive whole number?

<h1><u>Exercise 2</u></h1>

Note that there are the same number of ways to get 4 tails as there to get 4 heads. Explain why this is.
